langkit/tests/test_input_output.py

```diff
@@ -36,7 +36,7 @@ def texty(d):
 @pytest.mark.load
 def test_similarity(interactions):
     # default input col is "prompt" and output col is "response".
-    from langkit import input_output as lkio
+    from langkit import input_output as lkio  # noqa
```

Comment:
```suggestion
    from langkit import input_output
```

Reply:
thanks that is better, but we still get a warning wit this suggested change, lets do a follow up.